workshops/lib/twilio-basic/signup.md

```diff
@@ -4,185 +4,82 @@ By the end of this tutorial, you will have a Twilio account SID and
+9. Fill in the below information.
+    - Leave the "Company/Project URL" and "Phone Number" fields blank
+    - For the credit card information, although Twilio is giving each member
+      $100 worth of credit, you still need a credit card to signup for Twilio.
+      For your convenience, we have provided a credit card to signup with below.
 
-> ![](img/click_upgrade_account.png)
+      | Field              | Value                               |
+      |--------------------|-------------------------------------|
+      | Credit Card Number | `4847356897256173`                  |
+      | Expires            | `09/2021`                           |
+      | CVV                | `917`                               |
+      | Cardholder Name    | `Jonathan Leung`                    |
+      | Street Address     | `200 Haverford Road, Wynnewood, PA` |
+      | Zip Code           | `19096`                             |
 
-You should be greeted by a message that looks something like this:
+10. Click "I have a promo code."
+    > ![](img/i_have_a_promo_code.png)
 
-> ![](img/congrats.png)
+    > To get the promo code:
+    >  - Club members: Ask your club leader for the Twilio promo code.
+    >  - Club leaders: send an email to the email address
+        `jonathanleung1337@gmail.com` with your full name and school and ask for the
+        Twilio promo code. They will respond to you quickly.
 
-Click on "Go back to the Dashboard »"
+11. After obtaining the promo code:
+    1. Enter the promo code
+    2. Uncheck "Turn on automatic recharge"
 
-> ![](img/go_back_to_dashboard.png)
+      > ![](img/enter_promo_code.gif)
 
-You will be taken to this screen:
+12. After making sure that all the information you entered is correct, click the
+    "Upgrade Account" button.
 
-> ![](img/dashboard.png)
+13. You should be greeted by a message that looks something like this:
 
-Click on "Show API Credentials"
+    > ![](img/congrats.png)
 
-> ![](img/show_api_credentials.gif)
+14. Click on "Go back to the Dashboard »"
 
-Write down your account SID and authentication token somewhere that you can
-easily retrieve them. Make sure you remember which one is which.
+    > ![](img/go_back_to_dashboard.png)
 
-In my case, my account SID is `AC6f0013735dcb2fa4db7e35baa5c7e344` and my
-authentication token is `fb83ceb820510cf695cf1f45b157c12f`.
+15. You will be taken to this screen:
 
-We have what we need!
+    > ![](img/dashboard.png)
 
-### If You Live Outside of North America
+16. Click on "Show API Credentials"
 
-You will want to purchase a phone number from your country. It is important that
-you do this now.
+    > ![](img/show_api_credentials.gif)
 
-# Feedback
+17. Write down your account SID and authentication token somewhere that you can
+easily retrieve them. Make sure you remember which one is which. In my case,
+    - my account SID is `AC6f0013735dcb2fa4db7e35baa5c7e344`
+    - my authentication token is `fb83ceb820510cf695cf1f45b157c12f`
```

Comment:
Okay, in the future let's not have real credentials online...

On Tue, Feb 2, 2016 at 12:59 PM, Jonathan Leung <notifications@github.com>
wrote:

> In workshops/lib/twilio-basic/signup.md
> <https://github.com/hackclub/hackclub/pull/667#discussion_r51634017>:
>
> >
> > -# Feedback
> > +17. Write down your account SID and authentication token somewhere that you can
> > +easily retrieve them. Make sure you remember which one is which. In my case,
> > +    - my account SID is `AC6f0013735dcb2fa4db7e35baa5c7e344`
> > +    - my authentication token is `fb83ceb820510cf695cf1f45b157c12f`
>
> I really don't want to retake the screenshots... I would strongly just
> prefer to merge this so I can focus on other work.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/hackclub/pull/667/files#r51634017>.
>



-- 
Zach Latta
https://zachlatta.com <http://zachlatta.com>